.gitignore

```diff
@@ -110,5 +110,8 @@ package-lock.json
 # Docker environment config
 docker_environment_init.sh
 
+#Django environment variables
+democracylab_environment_variables.sh
```

Comment:
We do still want this file around as a sample.  To guard against accidental changing and committing, we can do something like add the following to the top of the .sh file to keep it from being run accidentally:

read -n 1 -r -s -p $'This is an example .sh file, and should not be run or changed unless new environment variables are added.  Press any key to exit\n'
exit 1

Reply:
Added a protection code to democracylab_environment_variables.sh and removed democracylab_environment_variables.sh
 from .gitignore.